rfc6962-bis.xml

```diff
@@ -613,44 +613,44 @@ TBSCertificate (in the case of a PrecertChainEntryV2), as described above.
 SignedCertificateTimestamp v2. Currently, no extensions are specified.
         </t>
       </section>
-
       <section title="Including the Signed Certificate Timestamp in the TLS Handshake">
         <t>
-The SCT data corresponding to at least one certificate in the chain from at
-least one log must be included in the TLS handshake, either by using
-an X509v3 certificate extension as described below, by using a TLS
-extension
-(Section 7.4.1.4 of <xref target='RFC5246'/>) 
-with type
-"signed_certificate_timestamp", or by using Online Certificate
-Status Protocol (OCSP) Stapling (also known as the "Certificate
-Status Request" TLS extension; see 
-<xref target='RFC6066'/>), where the OCSP response includes a non-critical
-extension with OID 1.3.6.1.4.1.11129.2.4.5 (see <xref target='RFC2560'/>) and
-body:
-
-
-        </t>
-        <figure>
-          <artwork>
-    SignedCertificateTimestampList ::= OCTET STRING</artwork></figure>
-        <t>
-          in the singleExtensions component of the SingleResponse pertaining to the end-entity certificate.
-        </t>
-        <t>
-          At least one SCT MUST be included. Server operators MAY include more than one SCT.
-        </t>
-        <t>
-          Similarly, a certification authority MAY submit a precertificate to more
-than one log, and all obtained SCTs can be directly embedded in the issued
-certificate, by encoding the SignedCertificateTimestampList structure as an
-ASN.1 OCTET STRING and inserting the resulting data in the TBSCertificate as a
-non-critical X.509v3 certificate extension (OID 1.3.6.1.4.1.11129.2.4.2). Upon
-receiving the certificate, clients can reconstruct the original TBSCertificate
-to verify the SCT signature.
+          The SCT data corresponding to at least one certificate in the chain
+from at least one log must be included in the TLS handshake by using one or more
+of the mechanisms listed below. Three mechanisms are provided because they have
+different tradeoffs. TLS clients MUST implement all three mechanisms. TLS
+servers MUST implement at least one of the three mechanisms.
```

Comment:
Perhaps something more like "TLS servers MUST present SCTs using at least on of the three..."